File: tests/fixtures/smoke/php/sample.php
<?php
// Smoke-tier fixture (BACK-431 Issue G) — PHP.
function validate($order) {
    if (empty($order)) {
        throw new InvalidArgumentException("empty order");
    }
    return $order;
}

function processOrder($order) {
    try {
        $result = validate($order);
        $count = 0;
        while ($count < 3) {
            $count++;
        }
        return strtoupper($result);
    } catch (Exception $e) {
        return null;
    }
}

function run($order) {
    return processOrder($order);
}

File: tests/fixtures/smoke/swift/sample.swift
// Smoke-tier fixture (BACK-431 Issue G) — Swift.
func validate(_ order: String?) throws -> String {
    guard let order = order else {
        throw ValidationError.empty
    }
    return order
}

func processOrder(_ order: String?) -> String? {
    do {
        let result = try validate(order)
        var count = 0
        while count < 3 {
            count += 1
        }
        return result.uppercased()
    } catch {
        return nil
    }
}

func run(_ order: String?) -> String? {
    return processOrder(order)
}
